packages/reports/src/Containers/open-positions.jsx

```diff
@@ -309,10 +310,28 @@ const OpenPositions = ({
     server_time,
     ...props
 }) => {
-    const [active_index, setActiveIndex] = React.useState(is_multiplier ? 1 : 0);
     // Tabs should be visible only when there is at least one active multiplier contract
     const [has_multiplier_contract, setMultiplierContract] = React.useState(false);
+    const [contract_type_value, setContractTypeValue] = React.useState('Options');
     const previous_active_positions = usePrevious(active_positions);
+    const contract_types = [
```

Comment:
`All` option is missing.

Reply:
`All` option not supposed to be in design. Seems it's a mistake their figma design. See thread for clarification.

https://deriv-group.slack.com/archives/C0YTBCXM1/p1674715536419229?thread_ts=1674713177.455879&cid=C0YTBCXM1